redux-form/lib/actions.d.ts

```diff
@@ -138,7 +134,3 @@ export function unregisterField(form: string, name: string): Action;
  * Resets the 'touched' flag for all the fields passed in.
  */
 export function untouch(form: string, ...fields: string[]): Action;
-
-export function updateSyncErrors(from: string, syncErrors: FormErrors<FormData>, error: any): Action;
-
-export function updateSyncWarnings(form: string, syncWarnings: FormWarnings<FormData>, warning: any): Action;
```

Comment:
In my experience with this, it's just missing from docs, not private. I don't see any history for that doc page where the functions are removed, they were just never there.

IMO, it's exported == it's public.